lib/node/nodes/routing-sequential.js

```diff
@@ -33,11 +31,11 @@ var routingSequentialQueryTemplate = Node.template([
     'SELECT',
     '  (route).duration,',
     '  (route).length,',
-    '  (route).the_geom',
+    '  (route).shape as the_geom',
```

Comment:
Yes. The returned custom type is 4326 as you can see [here](https://github.com/CartoDB/dataservices-api/blob/development/client/sql/16_custom_types.sql#L8)